src/interceptor/interceptable-http.ts

```diff
@@ -17,18 +17,19 @@
  limitations under the License.
  */
 import { HttpInterceptor } from './http.interceptor';
-import { Injectable } from '@angular/core';
+import { forwardRef, Inject, Injectable } from '@angular/core';
 import { Http, ConnectionBackend, RequestOptions, RequestOptionsArgs, Request, Response, Headers } from '@angular/http';
 import { Observable } from 'rxjs/Observable';
 
 @Injectable()
 export class InterceptableHttp extends Http {
+    // private interceptors: HttpInterceptor[];
     private firstInterceptor: HttpInterceptor;
 
     constructor(
         backend: ConnectionBackend,
         defaultOptions: RequestOptions,
-        interceptors: HttpInterceptor[]
+        @Inject(forwardRef(() => HttpInterceptor)) interceptors: HttpInterceptor[]
```

Comment:
using `@Inject` should be avoided in styleguide 🤔 
But we can merge this, another workaround 😢